temporalio/contrib/opentelemetry.py

```diff
@@ -125,11 +125,10 @@ def workflow_interceptor_class(
         :py:meth:`temporalio.worker.Interceptor.workflow_interceptor_class`.
         """
         # Set the externs needed
-        # TODO(cretz): MyPy works w/ spread kwargs instead of direct passing
         input.unsafe_extern_functions.update(
-            **_WorkflowExternFunctions(
-                __temporal_opentelemetry_completed_span=self._completed_workflow_span,
-            )
+            {
+                "__temporal_opentelemetry_completed_span": self._completed_workflow_span,
+            }
```

Comment:
What is the harm in using the constructor form of the TypedDict here instead of the dict literal with string literals?